static/js/stream_edit.js

```diff
@@ -292,9 +292,8 @@ exports.sort_but_pin_current_user_on_top = function (users) {
     }
 };
 
-function show_subscription_settings(sub_row) {
-    const stream_id = sub_row.data("stream-id");
-    const sub = stream_data.get_sub_by_id(stream_id);
+function show_subscription_settings(sub) {
+    const stream_id = sub.stream_id;
```

Comment:
The refactor to pass this a `sub` might be worth doing as a prep commit to simplify things here.